docs/contribute/developer-guide.md

```diff
@@ -131,6 +131,17 @@ Just open APITable's root directory with IDE.
 
 
 
+## How to contribute translations?
+
+We have two ways to improve the translation of APITable:
+
+1. Modify the source code and create a PR
```

Comment:
You can modify the markdown files in source code and create a PR directly.

Reply:
But there may still be json files that need to be translated in the future.